<launch>
    <!-- the bottom facing camera -->

    <!-- TODO: use quaternion for orientation -->
    <node pkg="tf2_ros" type="static_transform_publisher" name="bottom_camera_frame" args="0 0 0 -1.5707963 0 3.1415926 fcu bottom_camera_optical"/>

    <node pkg="cv_camera" type="cv_camera_node" name="bottom_camera">
        <param name="camera_info_url" value="file://$(find clever)/camera_info/fisheye_cam.yaml"/>
        <param name="frame_id" value="bottom_camera_optical"/>
        <param name="rate" value="30"/>
        <param name="image_width" value="320"/>
        <param name="image_height" value="240"/>
    </node>
</launch>
